nitrous.readme.md

```diff
@@ -0,0 +1,68 @@
+# Nitrous Brunch Quickstart
```

Comment:
I'll find out how moving these files affects the process internally, as at the moment we've been told to they need to be in the root of the Git project; should be okay though for the readme. Will get back to you on this.